embulk-core/src/main/java/org/embulk/spi/ReporterCloseable.java

```diff
@@ -0,0 +1,9 @@
+package org.embulk.spi;
+
+public interface ReporterCloseable
```

Comment:
Yea, that's correct. We don't need `ReporterCloseable` interface and I moved `AbstractReporter` with `org.embulk.spi.AbstractReporterImpl` 
on c141f7e7c0740f51deab1876e5b11a3550f80aa0 and aa96e14381142488440b7f9aff6200861f554ace